src/pages/staff/[user_id].tsx

```diff
@@ -20,19 +40,45 @@ const ViewParaPage = () => {
     }
   );
 
-  const unassignPara = trpc.case_manager.removePara.useMutation({
+  const buttonSX = {
+    "&:hover": {
+      background: "#3023B8",
+    },
+  };
+
+  const editMutation = trpc.case_manager.editPara.useMutation({
+    onSuccess: () => utils.para.getParaById.invalidate(),
+  });
+
+  const handleEditPara = (e: React.ChangeEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    const data = new FormData(e.currentTarget);
+
+    if (!para) {
+      return; // TODO: improve error handling
+    }
+    editMutation.mutate({
```

Comment:
🐧 might want 
```suggestion
    await editMutation.mutateAsync({
```

so the view doesn't change until the mutation succeeds